src/main/resources/db/changelog/changes/12-create-cart-items-table.yaml

```diff
@@ -0,0 +1,40 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-cart-items-table
+      author: ponomvrenko
+      changes:
+        - createTable:
+            tableName: cart_items
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

Comment:
when column is primary key constraint nullable is redundant, because primary key is unique + not null.